Plugins/Public/playercntl_plugin/AntiJumpDisconnect.cpp

```diff
@@ -26,18 +26,18 @@ namespace AntiJumpDisconnect
 {
 	struct INFO
 	{
-		bool bInWrapGate;
+		bool bInWarpGate;
 	};
 	static map<uint, INFO> mapInfo;
```

Comment:
Why have a map to a struct of single bool? Skip the struct part out or better document.